doc/getting_started.rst

```diff
@@ -64,7 +64,7 @@ Environment Variables
 WAM-Server needs at least the following environment variables:
 
 - CONFIG_PATH: Path to configuration file (mainly includes database configurations, see :ref:`configuration`)
-- DJANGO_SECRET_KEY: Set a secret key for *settings.py* (from *settings.py*: SECURITY WARNING: keep the secret key used in production secret!)
```

Comment:
It seemed to me that this was redundant :)

Reply:
you are right!
I changed code to use only env vars `CONFIG_PATH` and `WAM_APPS`.
My plan is to move `WAM_APPS` also into config file and only set env `CONFIG_PATH`